include/deal.II/base/derivative_form.h

```diff
@@ -149,7 +153,7 @@ class DerivativeForm
 
 private:
   /**
-   * Auxiliary function that computes (*this) * $T^{T}$
+   * Auxiliary function that computes [*this]  $T^{T}$
```

Comment:
I think this correction is lacking a `*`. I think it was correct the way it was.

Reply:
We argued in other places that we don't want to use asterisks in math formulas for multiplication of tensors and matrices. This one is awkward because of the occurrence of `this`. How about doing this better by changing
```suggestion
   * Auxiliary function that computes $AT^{T}$ where $A$ represents the current object
```
(I can't see right now what the next line is, so you may have to adjust the text)

Reply:
Fixed it.